.github/workflows/ci-zk-toolbox-reusable.yml

```
@@ -100,32 +103,30 @@ jobs:
           --prover-db-url=postgres://postgres:notsecurepassword@localhost:5432 \
           --prover-db-name=zksync_prover_localhost_custom_token \
           --port-offset 3000 \
-          --chain chain_custom_token
-
-      - name: Check Database
-        run: |
-          ci_run zk_supervisor database check-sqlx-data
+          --chain custom_token
 
       - name: Build test dependencies
         run: |
           ci_run zk_supervisor test build
 
       - name: Run servers
         run: |
-          ci_run zk_inception server --ignore-prerequisites --chain era &> server_rollup.log &
-          ci_run zk_inception server --ignore-prerequisites --chain chain_validium &> server_validium.log &
-          ci_run zk_inception server --ignore-prerequisites --chain chain_custom_token &> server_custom_token.log &
+          mkdir -p logs/server
+          ci_run zk_inception server --ignore-prerequisites --chain era &> logs/server/server_rollup.log &
+          ci_run zk_inception server --ignore-prerequisites --chain validium &> logs/server/validium.log &
+          ci_run zk_inception server --ignore-prerequisites --chain custom_token &> logs/server/custom_token.log &
           ci_run sleep 5
 
       - name: Run integration tests
         run: |
-          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain era &> integration_rollup.log &
+          mkdir -p logs/integration_tests
+          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain era &> logs/integration_tests/rollup.log &
           PID1=$!
 
-          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain chain_validium &> integration_validium.log &
+          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain validium &> logs/integration_tests/validium.log &
```

We could add a step like this:
```
- name: Create log directories
        run: |
          SERVER_LOGS_DIR=logs/server
          INTEGRATION_TESTS_LOGS_DIR=logs/integration_tests
          EXTERNAL_NODE_LOGS_DIR=logs/external_node

          mkdir -p $SERVER_LOGS_DIR
          mkdir -p $INTEGRATION_TESTS_LOGS_DIR
          mkdir -p $EXTERNAL_NODE_LOGS_DIR

          echo "SERVER_LOGS_DIR=$SERVER_LOGS_DIR" >> $GITHUB_ENV
          echo "INTEGRATION_TESTS_LOGS_DIR=$INTEGRATION_TESTS_LOGS_DIR" >> $GITHUB_ENV
          echo "EXTERNAL_NODE_LOGS_DIR=$EXTERNAL_NODE_LOGS_DIR" >> $GITHUB_ENV
```

```suggestion
          ci_run zk_supervisor test integration --no-deps --ignore-prerequisites --chain validium &> ${{ env.INTEGRATION_TESTS_LOGS_DIR }}/validium.log &
```